core/scripts/ocr2vrf/setup_dkg.go

```diff
@@ -155,7 +156,7 @@ func resetDatabase(client *cmd.Client, context *cli.Context, index int, database
 	helpers.PanicErr(client.ResetDatabase(context))
 }
 
-func newDKGSetupClient() *cmd.Client {
+func newSetupClient() *cmd.Client {
```

Comment:
`newSetupClient` and `setupOCR2VRFNodeFromClient` might be better in the util.go file now that they're being used by multiple files.

Reply:
Moved this and the other functions that are common between the two.